libr/bin/format/elf/elf.c

```diff
@@ -2118,8 +2196,26 @@ char* Elf_(r_bin_elf_get_head_flag)(ELFOBJ *bin) {
 		default :                   return strdup (" Unknown mips ISA");
 		}
 	}
-	//TODO: Fill other arch 
-	return strdup("unknown_flag");
+	return NULL;
+}
+
+char* Elf_(r_bin_elf_get_head_flag)(ELFOBJ *bin) {
+	char *head_flag = r_str_new("");
```

Comment:
We also have `RStrBuf` to create strings, if you want.

Reply:
okay~ thanks